sdk/tables/azure-data-tables/tests/test_table_async.py

```diff
@@ -154,10 +190,33 @@ async def test_query_tables_with_filter(self, resource_group, location, storage_
         # Assert
         self.assertIsNotNone(tables)
         self.assertEqual(len(tables), 1)
-        # self.assertEqual(tables[0].table_name, [table.table_name])
-        # table.delete_table()
+        for table_item in tables:
```

Comment:
Can you also include a specific check for date and api_version? I think just checking that thedy're not None and not empty are good (i.e. `assert table_item.api_version`)

Reply:
the `api_version` is not returned by the service unless the `full_metadata` option is specified, I added tests for the other two properties